pom.xml

```diff
@@ -94,6 +94,11 @@
             <version>RELEASE</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>commons-beanutils</groupId>
```

Comment:
Where do you use this dependency?

Reply:
in FieldMatchValidator.class

in isValid method:

    @Override
    public boolean isValid(Object value, ConstraintValidatorContext context) {
        try {
            final Object firstObj = BeanUtils.getProperty(value, firstFieldName);
            final Object secondObj = BeanUtils.getProperty(value, secondFieldName);

            return firstObj != null && firstObj.equals(secondObj);
        } catch (IllegalAccessException | InvocationTargetException | NoSuchMethodException e) {
            LOGGER.error("Error accessing properties in FieldMatchValidator", e);
            return false;
        }
    }

Reply:
refactored with BeanWrapperImpl class